scripts/run-integration-tests.sh

```diff
@@ -31,7 +40,12 @@ if [ "$TEST_CONFIG_EXISTS" == true ]; then
     # shellcheck disable=SC1091
 
     printf "\n\nExecuting integration tests against env %s\n" "$ENV_NAME"
-    pnpm run intTest --recursive --if-present -- --stage="$ENV_NAME"
+
+    if [ "$APPSTREAM_ENABLED" == "AppStream" ]; then
```

Comment:
This flag should be `APPSTREAM_EGRESS_ENABLED`, right?

Reply:
sorry about that, good catch. Updated.